test/system/version_test.go

```diff
@@ -0,0 +1,50 @@
+package system
+
+import (
+	"fmt"
+	"os"
+	"regexp"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("Podman version test", func() {
```

Comment:
I would like to see a more verbose description that explains the purpose of this set of tests.

Reply:
The details description is in following lines inside "It"

Reply:
How about:

```go
var _ = Describe("Verify version sub-command positive and negative behavior with different args/flags", func() {
```